Sloth.Web/Controllers/TransactionsController.cs

```diff
@@ -19,13 +20,38 @@ public TransactionsController(ApplicationUserManager userManager, SlothDbContext
         // GET: /<controller>/
         public async Task<IActionResult> Index()
         {
+            return await Filtered(DateTime.Now.AddMonths(-1), DateTime.Now);
+        }
+
+        public async Task<IActionResult> Filtered(DateTime @from, DateTime to)
+        {
+            var fromUtc = DateTime.SpecifyKind(@from, DateTimeKind.Local).ToUniversalTime().Date;
+            var throughUtc = DateTime.SpecifyKind(to, DateTimeKind.Local).ToUniversalTime().AddDays(1).Date;
+
+            if (fromUtc > DateTime.UtcNow)
+            {
+                return BadRequest("From cannot be a future date.");
```

Comment:
Don't do this. Instead add a Message (ViewBag, Temp Data) In the past we have added this to the SuperController.
Then either set the date to something valid or just return the results which would be empty.

Reply:
This was based on another endpoint `CreateReversal(string id)` in the controller that does this. Should that be corrected? Maybe open an issue to correct those throughout the solution?

Reply:
The create reversal is doing a post to change data, so in that case if something unexpected happens (not because of user input) we want an error. At least I believe that was John's logic.